docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -30,124 +30,28 @@ This guide assumes that you have:
-    rules:
-      - resources: ['access_request']
-        verbs: ['list', 'read']
-      - resources: ['access_plugin_data']
-        verbs: ['update']
-```
-
-Here and below follow along and create yaml resources using `tctl create -f`:
-
-```
-$ tctl create -f access.yaml
-```
-
-<Admonition type="tip">
-  If you're using other plugins, you might want to create different users and roles for different plugins
-</Admonition>
+### Create User and Role for access
+(!docs/pages/includes/plugins/rbac.mdx!)
 
-#### Export access-plugin Certificate
+### Export access-plugin Certificate
+(!docs/pages/includes/plugins/identity-export.mdx!)
 
-Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign). You have the option of connecting through the proxy or auth server.  Teleport Cloud must use the proxy server.
-
-<Tabs>
-  <TabItem label="OpenSource, Enterprise" scope={["oss","enterprise"]}>
-```code
-$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=2190h
-# ...
-```
-
-The above sequence should result in three PEM encoded files being generated: auth.crt, auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll reference the auth.crt, auth.key, and auth.cas files later when [configuring the plugins](#configuring-teleport-slack).
-</TabItem>
-  <TabItem label="Cloud" scope={["cloud"]}>
-```code
-$ tctl auth sign --user=access-plugin --out=auth.pem --ttl=2190h
-# ...
-```
-
-The above sequence should result in one PEM encoded file being generated: auth.pem.  We'll reference the auth.pem file later when [configuring the plugins](#configuring-teleport-slack).
-
-  </TabItem>
-
-</Tabs>
-
-
-
-<Admonition
-  type="note"
-  title="Certificate Lifetime"
->
-  By default, [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign) produces certificates with a relatively short lifetime. For production deployments, the `--ttl` flag can be used to ensure a more practical certificate lifetime. `--ttl=8760h` exports a 1 year token
-</Admonition>
+We'll reference these files later when [configuring the plugins](#configuring-teleport-slack).
 
 ### Create Slack App
 
-We'll create a new Slack app and setup auth tokens and callback URLs, so that Slack knows how to notify the Teleport plugin when Approve / Deny buttons are clicked.
+We'll create a new Slack app and set up auth tokens and callback URLs, so that Slack knows how to notify the Teleport plugin when Approve / Deny buttons are clicked.
 
 You'll need to:
 
 1. Create a new app, pick a name and select a workspace it belongs to.
-2. Add OAuth Scopes. This is required by Slack for the app to be installed — we'll only need a single scope to post messages to your Slack account.
-3. Obtain OAuth token
+2. Add OAuth Scopes. This is required by Slack for the app to be installed—we'll only need a single scope to post messages to your Slack account.
```

Comment:
Is this actually correct? White space around the `-` makes more sense to me, as this isn't a hyphenated term.